src/legacy/ui/public/agg_types/controls/raw_json.tsx

```diff
@@ -68,6 +69,7 @@ function RawJsonParamEditor({
         onChange={onChange}
         rows={2}
         fullWidth={true}
+        onBlur={setTouched}
```

Comment:
Don't we want the same for the string control? Otherwise validation there wouldn't follow that scheme (which could happen if you have a string parameter which is required?

Reply:
The string control wasn't required initially https://github.com/elastic/kibana/pull/32881/files?file-filters%5B%5D=.html#diff-da2faab3625949023e13a8d471b1fa64L1 . And there is no any validation for string controls. So it's not necessary for adding validation to the string control.

Reply:
In case string control will be used with `required` config in the future, I added validation.